Extensions/drafts_plus.js

```diff
@@ -259,8 +259,12 @@ XKit.extensions.drafts_plus = new Object({
 			GM_xmlhttpRequest({
 				method: "POST",
 				url: "http://www.tumblr.com/svc/post/delete",
-				data: JSON.stringify(m_object),
-				json: true,
+				headers: {
+					"Content-Type": "application/x-www-form-urlencoded",
+					"X-tumblr-form-key": XKit.interface.form_key(),
+				},
+				data: "post_id=" + m_object.post_id + "&channel_id=" + m_object.channel_id,
```

Comment:
That works too. I was following the pattern already existing in the other calls; I'll update those as well.